packages/components-react/src/checkbox/Checkbox.css

```diff
@@ -25,3 +31,26 @@
 .lux-checkbox--disabled {
   cursor: not-allowed;
 }
+
+.lux-checkbox--with-target {
+  display: inline-grid;
+  grid-template-areas: "input";
+  place-content: center center;
+}
+
+.lux-checkbox--with-target::before,
```

Comment:
ℹ️ de `::before` vergroot de click target, de `::after` zet de achtergrond, maar omdat die normaal voor de checkbox komt, krijgt die `z-index: -1` mee, alleen dan werkt het niet meer als click target..., vandaar de opsplitsing.

Reply:
De implicaties van deze code begrijp ik. withTarget zou inderdaad een keuze moeten zijn. Enige wat mij opvalt is als withTarget=false de background color van de gehele field oplicht, maar alleen als je op de checkbox of de label hovered. Even met UX checken of dit gewenst gedrag is.